src/components/ui-shell/_ui-shell.scss

```diff
@@ -4,4 +4,4 @@
 @import 'header';
 @import 'product-switcher';
 @import 'side-nav';
-@import 'nav';
+@import 'popup-nav';
```

Comment:
For consistency, should we call this navigation menu? Just thinking about this sketch artboard:

![image](https://user-images.githubusercontent.com/3901764/48630558-2de80200-e982-11e8-82fa-dade59687072.png)

Reply:
I renamed the components to `navigation-menu` and `navigation-menu-panel`